Drop-In/src/test/java/com/braintreepayments/api/dropin/DropInActivityUnitTest.java

```diff
@@ -548,4 +548,4 @@ private void assertExceptionIsReturned(String analyticsEvent, Exception exceptio
         assertEquals(exception.getClass(), actualException.getClass());
         assertEquals(exception.getMessage(), actualException.getMessage());
     }
-}
+}
```

Comment:
This test verifies that when Android Pay is not selectable in the payment method selector, when we  `disableAndroidPay` 

Add this test here:

    @Test
    public void onConfigurationFetched_whenAndroidPayDisabledClientSide_doesNotShowAndroidPay() {
        BraintreeUnitTestHttpClient httpClient = new BraintreeUnitTestHttpClient()
                .configuration(new TestConfigurationBuilder()
                        .androidPay(new TestConfigurationBuilder.TestAndroidPayConfigurationBuilder())
                        .build());
        mActivity.setDropInRequest(new DropInRequest()
                .disableAndroidPay()
                .tokenizationKey(TOKENIZATION_KEY));
        setup(httpClient);

        assertEquals(0, ((ListView) mActivity.findViewById(R.id.bt_supported_payment_methods)).getAdapter().getCount());
    }